### ISSUE TITLE
new tweets between last refresh and a new post are lost

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#126]
```
What steps will reproduce the problem?
1. refresh new tweets
2. wait 1'
3. post a new tweet

What is the expected output? What do you see instead?
New tweets between the last refresh and my tweet are totally lost

What version of the product are you using? On what operating system?
pino-0.2.1 at fedora 12 (both i686 and x64)

Please provide any additional information below.

```

Original issue reported on code.google.com by `comzer...@chimeres.gr` on 10 Mar 2010 at 12:43

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 10 Mar 2010 at 4:59
- Changed state: **Accepted**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Issue 137 has been merged into this issue.
```

Original comment by `tro...@gmail.com` on 16 Mar 2010 at 10:53

**Comment by GoogleCodeExporter** [comment]
```
Closed by 27a3476870 revision.
```

Original comment by `tro...@gmail.com` on 25 Mar 2010 at 12:07
- Changed state: **Fixed**
